### PR TITLE
COMP: Fix typo (compile error) in `PhilipsRECImageIO` constructor

### PR DESCRIPTION
**Pull request by N-Dekker**
Fixed a compile error, which appeared when enabling `Module_ITKIOPhilipsREC`:

> itkPhilipsRECImageIO.cxx(390,3): error C3861: 'iff': identifier not found

This obvious typo was introduced with pull request https://github.com/InsightSoftwareConsortium/ITK/pull/3664 commit 2b1b5312fadf1a156882d06dc2e7c6b2a2290044 "STYLE: Remove space between class and member names (follow-up)", and reported by Bradley Lowekamp (@blowekamp).

### CODE REVIEW

**Review by dzenanz — APPROVED** [review]

**Comment by dzenanz** [comment]
Preferably merge today.

**Comment by N-Dekker** [comment]
> Preferably merge today.

@dzenanz Do you think we should still wait for the CI? It did not notice the typo in the first place, as Module_ITKIOPhilipsREC is off by default.

**Comment by dzenanz** [comment]
CI doesn't seem to be running. Could you force-push to try to re-trigger it? I would prefer for CI to finish, at least cxx tests. Python packaging takes longer, and if you need to go to sleep before it finishes, merge 😄

**Pull request merged**
